source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -500,6 +500,71 @@ network provider on the host (unless implemented by the user).
 * get_caps(caps, properties)
 * get_stats(stats, properties)
 
+## Using OSP8 as an external provider
+This feature aims to integrate with any external network provider. We have tested it with OSP8's Neutron using packstack installation, with the following steps and workarounds.
+
+#### Install OSP8 with packstack on Controller, Compute and Networker(the host that will run the network services)
+Please NOTE: This is how we tested, installed and integrated with oVirt(3.6 and 4.0) step by step.
+Some of the configurations, answers in the answer file and workarounds, may be relevant only to our specific or to some other specific use cases.
```

Comment:
Delete:Some of the configurations, answers in the answer file and workarounds, may be relevant only to our specific or to some other specific use cases.

Instead we should provide a generic enough description later on.